app/views/hyrax/dashboard/sidebar/_repository_content.html.erb

```diff
@@ -12,8 +12,23 @@
     <span class="fa fa-file" aria-hidden="true"></span> <span class="sidebar-action-text"><%= t('hyrax.admin.sidebar.works') %></span>
   <% end %>
 
-  <% if can? :index, Hyrax::BatchIngest::Batch %>
-    <%= menu.nav_link(hyrax_batch_ingest.batches_path) do %>
-      <span class="fa fa-table" aria-hidden="true"></span> <span class="sidebar-action-text"><%= 'Batches' %></span>
-    <% end %>
+  <%# using bulkrax instead %>
+    <%# if can? :index, Hyrax::BatchIngest::Batch %>
+      <%# menu.nav_link(hyrax_batch_ingest.batches_path) do %>
+        <%
+=begin%>
+ <span class="fa fa-table" aria-hidden="true"></span> <span class="sidebar-action-text"><% 'Batches' %></span>
+<%
+=end%>
+      <%# end %>
+    <%# end %>
```

Comment:
since I removed references to the bulkrax ENV variable, this is now just commented out. if you're sure you won't need it again, it can be removed.